packages/shared/src/components/feeds/FeedNav.tsx

```diff
@@ -28,12 +28,21 @@ enum FeedNavTab {
   Discussions = 'Discussions',
 }
 
+const urlToTab: Record<string, FeedNavTab> = {
+  '/': FeedNavTab.ForYou,
+  '/popular': FeedNavTab.Popular,
+  '/upvoted': FeedNavTab.MostUpvoted,
+  '/discussed': FeedNavTab.Discussions,
+  '/bookmarks': FeedNavTab.Bookmarks,
+  '/history': FeedNavTab.History,
+};
```

Comment:
We can probably use this in `<Tab url={} />` as well.

Reply:
good idea, let me try :)

Reply:
On second though, you would need inverse of it. Non blocking from my end.